ui/apps/platform/package.json

```diff
@@ -65,7 +65,6 @@
         "react-feather": "^2.0.9",
         "react-helmet": "^6.1.0",
         "react-modal": "^3.12.1",
-        "react-monaco-editor": "^0.51.0",
```

Comment:
No longer a required peer dep, as of https://github.com/patternfly/patternfly-react/pull/9842